### PR TITLE
Add glob matching for the tcp+sni proxy

### LINKED ISSUE #462: Add glob support to tcp+sni
We'd love to have glob matching for the tcp+sni requests. 

I'll make a PR for this if I can.

### PR DESCRIPTION
**Pull request by jorgemarey**
Fixes #462

### CODE REVIEW

**Comment by magiconair** [comment]
Can you rebase and test this on top of #457 since I want to merge that first.

**Comment by jorgemarey** [comment]
Ok. Will do!

**Comment by magiconair** [comment]
I'll try to roll this into the next release.

**Comment by jorgemarey** [comment]
OK. Great!

File: route/table.go [hunk 1]
```diff
@@ -337,6 +343,16 @@ func (t Table) Lookup(req *http.Request, trace string, pick picker, match matche
 	return target
 }
 
+func (t Table) LookupHostGlob(host string, pick picker) (target *Target) {
```
Comment (magiconair) [inline route/table.go]: This isn't really a glob matching function since glob matching is already enabled. This only finds https targets.

**Comment by nathanejohnson** [comment]
Closing due to age

**Pull request closed without merging**